common/src/test/scala/uk/ac/wellcome/platform/archive/common/storage/services/S3StreamableTest.scala

```diff
@@ -0,0 +1,70 @@
+package uk.ac.wellcome.platform.archive.common.storage.services
+
+import java.nio.file.Paths
+
+import org.scalatest.concurrent.ScalaFutures
+import org.scalatest.{FunSpec, TryValues}
+import uk.ac.wellcome.platform.archive.common.fixtures.RandomThings
+import uk.ac.wellcome.platform.archive.common.storage.Resolvable
+import uk.ac.wellcome.storage.ObjectLocation
+import uk.ac.wellcome.storage.fixtures.S3
+
+class S3StreamableTest
+    extends FunSpec
+    with S3
+    with ScalaFutures
+    with TryValues
+    with RandomThings {
+
+  implicit val _s3Client = s3Client
```

Comment:
nit: the S3 client provided by the `S3` fixture has been implicit for a while, so I don't think you need this.